retis/src/module/nft/bpf/nft.bpf.c

```diff
@@ -7,6 +7,7 @@
 #define VERD_SCALE (NFT_RETURN * -1)
 #define ALLOWED_VERDICTS(verd, mask) (1 << (verd + VERD_SCALE) & mask)
 #define NFT_NAME_SIZE 128
+__binding const u32 nft_name_size = NFT_NAME_SIZE;
```

Comment:
Please add a comment here on why we need this construction, like the other. Alternatively we could use a `BINDING_DEF` macro that does the trick somehow.

Also, it's not used, do we need it in the UAPI? Only `log_events_max` is used (the macro could still be nice here). It seems array sizes are not useful to be shared in the UAPI.

Reply:
>  Alternatively we could use a BINDING_DEF macro that does the trick somehow.

I'm thinking something like (highly untested):

```c
#define BINDING_DEF(def, val) enum enum_ ## def { def = val } __binding
```

Reply:
> Please add a comment here on why we need this construction, like the other. Alternatively we could use a `BINDING_DEF` macro that does the trick somehow.
> 
> Also, it's not used, do we need it in the UAPI? Only `log_events_max` is used (the macro could still be nice here). It seems array sizes are not useful to be shared in the UAPI.

I guess this was used when there was the default trait implementation.
If it's not used anymore, it's better to remove it.
No need to carry unneeded things

Reply:
The above could still be used for `log_events_max`.

Reply:
my plan was to find a nice way to avoid the usage of types bigger than 8 bytes for alignment requirements.
This should happen through static assertion at compile time, meaning we might end up wrapping event-binding declarations as well.
If you agree, we can add this wrapper and the above in a follow-up as they are both not blockers for the time being.